### DISCUSSION TITLE
Error System.Net.CookieException on new downloads

### DISCUSSION THREAD

**Discussion by tccadam** [Tyrrrz/YoutubeDownloader#410] [General]
Hello, 

Anyone else getting this error as of today?  I used it earlier today and it worked, now any new downloads from 2 different PCs are giving me the same error shown below:
 
![image](https://github.com/Tyrrrz/YoutubeDownloader/assets/64762525/e55ad9ae-990f-4cc0-a68a-cca16fb9626f)

Maybe this is a temporary problem on youtube's side?

**Comment by GrimJackIII** [comment]
I am also having this issue as of today.

**Comment by coollao3** [comment]
I have the same issue.

**Comment by ouyangyu03** [comment]
i also have the same issue. i dont know how to solve it :(
![1](https://github.com/Tyrrrz/YoutubeDownloader/assets/83320219/6adf19cd-512b-4d46-8aa7-e3a154e57b59)

**Comment by crolyss** [comment]
On a hunch I tried logging in to my google account, and it started working again. I noted this in the issue page:

https://github.com/Tyrrrz/YoutubeDownloader/issues/411#issuecomment-1898424512
